packages/stencil/components/tabs/.stencil/screenshot-cache.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
.stencil should not be pushed

Reply:
In .stencil folder there is screenshot_79e7d9324787763c81039cd039d41aaf.png.js, we don't need to push this file ?

Reply:
👍